uniffi_bindgen/src/interface/function.rs

```diff
@@ -90,6 +90,21 @@ impl Function {
         self.ffi_func.return_type = self.return_type.as_ref().map(|rt| rt.into());
         Ok(())
     }
+
+    // Intentionally exactly the same as the Method version
```

Comment:
Aye, it'd be cool to factor some of the duplication out between `Function`/`Method`/`Constructor`, maybe as a `Callable` trait with default implementations. But, not for this PR.